engines/asm/e_padlock-x86.pl

```diff
@@ -113,6 +119,27 @@
 	&ret	();
 &function_end_B("padlock_capability")
 
+&function_begin_B("get_cpu_fms");
+    &push   ("ebx");
```

Comment:
There is no CFI Stack info here.
For linux and x86_64 we do always have correct stack unwind info.
(For Windows 64bit  I would not expect that to work what is there but I dont care for that OS.)

Reply:
or is this x86 only?

Reply:
in that case it is okay to not have stack unwind info (that means,
of course of someone has plenty of spare time, he could start to
add correct CFI infos everywhere)

Reply:
Thank you very much. 
But could you give me some information about CFI Stack info?  
I don't know what is CFI Stack info...

Reply:
The CFI information gives frame info for the debugger and stack walks.

Please look at other x86_64 frame infos how that should look like.
I am not sure if the windows frame infos work or not.
but the linux frame info should really work everywhere except in padlock.
at the very minimum we need the .cfi_startproc and .cfi_endproc.

You can use gdb to step in the procedures and show the stack frame
with "bt", however gdb is able to work around broken cfi infos,
and I used this https://github.com/openssl/openssl/pull/8706#issuecomment-605393761
to make the speed command print stack traces without gdb,
that revealed a lot of broken cfi infos.  Fixing those is good for
C++ exception handling code for instance, as this might otherwise crash
randomly.